extern/storage-sealing/commit_batch.go

```diff
@@ -452,24 +456,43 @@ func (b *CommitBatcher) Stop(ctx context.Context) error {
 	}
 }
 
-func getSectorDeadline(curEpoch abi.ChainEpoch, si SectorInfo) time.Time {
-	deadlineEpoch := si.TicketEpoch + policy.MaxPreCommitRandomnessLookback
+func (b *CommitBatcher) getCommitCutoff(si SectorInfo) (time.Time, error) {
+	tok, curEpoch, err := b.api.ChainHead(b.mctx)
+	if err != nil {
+		log.Errorf("getting chain head: %s", err)
+		return time.Now(), nil
+	}
+
+	nv, err := b.api.StateNetworkVersion(b.mctx, tok)
+	if err != nil {
+		log.Errorf("getting network version: %s", err)
+		return time.Now(), err
```

Comment:
I'd just wrap these with xerrors.Errorf, and drop the log - errors returned from AddCommit are saved in sector logs, which makes it much easier to debug individual sectors

Reply:
seems reasonable